consultation_analyser/consultations/models.py

```diff
@@ -71,9 +71,8 @@ class Meta(UUIDPrimaryKeyModel.Meta, TimeStampedModel.Meta):
 
 
 class Theme(UUIDPrimaryKeyModel, TimeStampedModel):
-    # Label summarises a topic from a topic model for a given question
-    label = models.CharField(max_length=256, blank=True)
-    summary = models.TextField(blank=True)
+    short_description = models.TextField(blank=True)
```

Comment:
Good rename.

Do we want it to ever be blank?

Reply:
Yes, it will be blank - when we've generated topics but haven't yet run the LLM summariser.